app/indexers/concerns/iiif_print/child_indexer.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module IiifPrint
+  module ChildIndexer
+    extend ActiveSupport::Concern
+    include IiifPrint::IiifPrintBehavior
```

Comment:
since it's already in the `IiifPrint` namespace, would it be better to just use..?
```rb
include IiifPrintBehavior
```

Reply:
good catch!